components/elm/bld/namelist_files/use_cases/20thC_CMIP6_transient.xml

```diff
@@ -45,6 +45,10 @@
 
 <fsurdat hgrid="r05">lnd/clm2/surfdata_map/surfdata_0.5x0.5_simyr1850_c200609_with_TOP.nc</fsurdat>
 
+<!-- Hack to override files for ne1024 for SCREAM, even though the sim years do not match up -->
```

Comment:
@brhillman , same here:

```
commit 3e74f881fa314a16c8c8d97a066f29b06fe511e3
Author: Benjamin Hillman <bhillma@sandia.gov>
Date:   Tue Jan 16 02:30:15 2024 -0600

    Add hacks to elm use cases to use correct ne1024 land data
```

Reply:
These were added to get the decadal run to configure. Only affects ne1024.